spec.md

```diff
@@ -1802,6 +1802,8 @@ The CO MUST implement the specified error recovery behavior when it encounters t
 A Controller plugin MUST implement this RPC call if plugin has `EXPAND_VOLUME` controller capability.
 This RPC allows the CO to expand the size of a volume.
 
+This operation MUST be idempotent. If a volume corresponding to the specified volume ID is larger than or equal to the capacity requirements after expansion, the plugin MUST reply 0 OK.
```

Comment:
```suggestion
This operation MUST be idempotent. If a volume corresponding to the specified volume ID is larger than or equal to the capacity requirements after expansion, the plugin SHOULD reply 0 OK.
```

... since we need backwards compatibility for plugins that don't already implement this.

Reply:
Thanks for your review. I have updated this PR.